tests/test_integration.py

```diff
@@ -118,3 +109,34 @@ def test_full_app(
     assert "No bug - Tagging" in tag_log
     assert "FIREFOX_128_0esr_RELEASE" in tag_log
     assert hg_rev(hg_remote_repo_path, destination_branch) in tag_log
+
+
+@pytest.mark.skipif(NO_RABBITMQ, reason="This test doesn't work without rabbitMq")
+def test_no_duplicated_ack_messages(
+    test_config: Config,
+    get_payload: Callable,
+) -> None:
+    """This tests checkes that a long-running message is not processed more than
+    once.
```

Comment:
nit: First line of doc string should be complete summary.

(nit because this is a test)